test/e2e/gateway/utils.go

```diff
@@ -15,18 +15,22 @@ import (
 )
 
 // OptEnableMeshMTLS is a Kuma deployment option that enables mTLS on the default Mesh.
-var OptEnableMeshMTLS = framework.WithMeshUpdate(
-	"default",
-	func(mesh *mesh_proto.Mesh) *mesh_proto.Mesh {
-		mesh.Mtls = &mesh_proto.Mesh_Mtls{
-			EnabledBackend: "builtin",
-			Backends: []*mesh_proto.CertificateAuthorityBackend{
-				{Name: "builtin", Type: "builtin"},
-			},
-		}
-		return mesh
-	},
-)
+func enableMeshMTLS(mesh string) framework.KumaDeploymentOption {
```

Comment:
how about using `MTLSMeshKubernetes` instead

Reply:
working on it